provider/datagen/src/transform/cldr/cldr_serde/currency_data.rs

```diff
@@ -16,19 +16,15 @@ type ISOCode = TinyAsciiStr<3>;
 #[derive(PartialEq, Debug, Deserialize)]
 pub struct RoundingModes {
     #[serde(rename = "_rounding")]
-    // #[serde(deserialize_with = "serde_aux::prelude::deserialize_option_number_from_string")]
     pub rounding: Option<String>,
```

Comment:
Nit (can fix later): It would be nice if we could get the `deserialize_option_number_from_string` working. Later on you'll need to parse these to an integer, and it's better to handle that at deserialization time.

Reply:
I added a todo